composer.lock

```diff
@@ -3814,16 +3866,16 @@
         },
         {
             "name": "neos/content-repository",
-            "version": "4.3.6",
+            "version": "4.3.7",
```

Comment:
@davidspiola You did a composer update for everything ;)
So also Neos Update included.

Reply:
is this good or bad ?!